src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -143,6 +143,15 @@ const HoloFuelDnaInterface = {
     }
   },
   transactions: {
+    getPending: async transactionId => {
```

Comment:
Do we want this endpoint to *only* serve as a fetch for a single pending transaction?  If so, could we rename this function to reflect that specificity? (ie: 'getSinglePending')

Reply:
the convention we've been using is `get` endpoints return a single object, and `all` endpoints return a list.